README.md

```diff
@@ -1,2 +1,58 @@
 # cql-exec-fhir-mongo
-A FHIR data provider for the JavaScript CQL Execution project
+
+This project establishes a FHIR-based data source module for use with the CQL Execution Engine. Currently, FHIR 1.0.2 (DSTU2), FHIR 3.0.0 (STU3), FHIR 4.0.0 ,and FHIR 4.0.1 (R4) are supported.
```

Comment:
Can we add just a blurb here about it's a FHIR-based data sources that interacts with a mongo database?

I also don't think we need to list the FHIR versions. We should only do R4 anyway